pkg/fanal/artifact/local/fs_test.go

```diff
@@ -59,9 +59,12 @@ func TestArtifact_Inspect(t *testing.T) {
 								FilePath: "lib/apk/db/installed",
 								Packages: types.Packages{
 									{
-										ID:         "musl@1.1.24-r2",
-										Name:       "musl",
-										Version:    "1.1.24-r2",
+										ID:      "musl@1.1.24-r2",
+										Name:    "musl",
+										Version: "1.1.24-r2",
+										Identifier: types.PkgIdentifier{
+											PURL: "pkg:apk/musl@1.1.24-r2?arch=x86_64",
```

Comment:
as for image test - correct purls in this test file.

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/19fbf7b6f7d528e44d1cf720f78727c2956b4e27